rwgame/debugstate.cpp

```diff
@@ -435,12 +432,29 @@ void DebugState::spawnVehicle(unsigned int id)
 	glm::vec3 fwd = ch->rotation * glm::vec3(0.f, 1.f, 0.f);
 
 	glm::vec3 hit, normal;
-	if(game->hitWorldRay(ch->position + (fwd * 5.f), {0.f, 0.f, -2.f}, hit, normal)) {
+	if(game->hitWorldRay(ch->position + (fwd * 10.f), {0.f, 0.f, -2.f}, hit, normal)) {
```

Comment:
Can you use the code from SPAWN_VEHICLE?
([My spawner commit also had the improved spawnVehicle - you can just steal that](https://github.com/JayFoxRox/openrw/pull/11/commits/8f4cc97f149f4ca6d57ee16057f487ee78a7c244#diff-bd6ee77bba6402560d5fb3194f3760adR345))

The rotation of the vehicle will be different so that it's easier / faster to enter the car or place it in narrow spots.